package.json

```diff
@@ -4,18 +4,18 @@
   "private": true,
   "cracoConfig": "./craco.config.js",
   "dependencies": {
-    "@swagger-api/swagger-editor": "^5.0.0-alpha.44",
+    "@swagger-api/swagger-editor": "^5.0.0-alpha.45",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "node-polyfill-webpack-plugin": "^2.0.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^17.0.2",
```

Comment:
SwaggerEditor@5 only works on `react@17`.  Unfortunately it doesn't yet work on `react@18`.